src/lib/stores/marketplace.ts

```diff
@@ -0,0 +1,686 @@
+                value: '',
+                placeholder: 'ghp_1...',
+                required: true
+            },
+            {
+                name: 'GITHUB_WEBHOOK_SECRET',
+                description: `The secret used to verify that the webhook request comes from GitHub. <a class="u-bold" target="_blank" href="https://docs.github.com/en/developers/webhooks-and-events/securing-your-webhooks">Learn More</a>`,
+                value: '',
+                placeholder: 'd1efb...aec35',
+                required: true
+            }
+        ]
+    },
+    {
+        icon: 'icon-bookmark',
+        id: 'url-shortener',
+        name: 'URL Shortener',
+        tagline: 'Generate URL with short ID and redirect to the original URL when visited.',
+        permissions: ['any'],
+        events: [],
+        cron: '',
+        timeout: 15,
+        runtimes: [
+            {
+                name: 'node-18.0',
+                commands: 'npm install',
+                entrypoint: 'src/main.js',
+                providerRootDirectory: 'node/url-shortener'
+            }
+        ],
+        instructions: `For documentation and instructions check out [README.md](https://github.com/appwrite/templates/tree/main/node/url-shortener) file.`,
+        vcsProvider: 'github',
+        providerRepositoryId: 'templates',
+        providerOwner: 'appwrite',
+        providerBranch: 'main',
+        variables: [
+            {
+                name: 'APPWRITE_API_KEY',
+                description: `The API Key to talk to Appwrite backend APIs. <a class="u-bold" target="_blank" href="https://appwrite.io/docs/getting-started-for-server">Learn More</a>`,
+                value: '',
+                placeholder: 'd1efb...aec35',
+                required: true
+            },
+            {
+                name: 'APPWRITE_ENDPOINT',
+                description: `The URL endpoint of the Appwrite server. <a class="u-bold" target="_blank" href="https://appwrite.io/docs/getting-started-for-server">Learn More</a>`,
+                value: 'https://cloud.appwrite.io/v1',
+                placeholder: 'https://cloud.appwrite.io/v1',
+                required: false
+            },
+            {
+                name: 'APPWRITE_DATABASE_ID',
+                description: `The ID of the database to store the short URLs. <a class="u-bold" target="_blank" href="https://appwrite.io/docs/databases">Learn More</a>`,
+                value: 'urlShortener',
+                placeholder: 'urlShortener',
+                required: false
+            },
+            {
+                name: 'APPWRITE_COLLECTION_ID',
+                description: `The ID of the collection to store the short URLs. <a class="u-bold" target="_blank" href="https://appwrite.io/docs/collections">Learn More</a>`,
```

Comment:
```suggestion
                description: `The ID of the collection to store the short URLs. <a class="u-bold" target="_blank" href="https://appwrite.io/docs/collections">Learn more</a>.`,
```

Reply:
You gotta link to the readme here and add directions.

Don't think this link helps @Meldiron